.github/workflows/docs.yml

```diff
@@ -55,8 +53,16 @@ jobs:
       - name: Execute MDBook
         run: make mdbook-build
       - name: Build rustdoc
-        run: make rustdoc
+        run: |- 
+          rustup toolchain install $toolchain
+          cargo +$toolchain doc --workspace --no-deps
         env:
+          # When you visit https://chainsafe.github.io/forest/rustdoc you get an index.html
+          # listing all crates that are documented (copied out below).
+          # This isn't included by default, so we use the unstable `--enable-index-page` option
+          # https://doc.rust-lang.org/nightly/rustdoc/unstable-features.html#--index-page-provide-a-top-level-landing-page-for-docs
+          toolchain: nightly-2023-04-19
+          RUSTDOCFLAGS: -Dwarnings -Zunstable-options --enable-index-page
```

Comment:
Any particular reason for moving it outside of the Makefile? I find it helpful to have the targets available locally, especially in case of a CI failure.

Reply:
having `rustup install toolchain` in a makefile felt a bit smelly, but I suppose it's no different to the standard `make deps`. I'll move